src/getHumanAge.test.js

```diff
@@ -1,8 +1,34 @@
+'use strict';
+
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be 0 in human age', () => {
+    expect(getHumanAge(0, 0)).toStrictEqual([0, 0]);
+  });
+
+  test('should be 0, 0 in human age when cat age 14,dog age 14', () => {
```

Comment:
```suggestion
  test('should be [0, 0] in human age when cat age is 14, dog age is 14', () => {
```

because now it seems like the function returns a number